src/django/api/tests.py

```diff
@@ -61,90 +61,130 @@ def setUpTestData(cls):
         cls.boundary_4 = Boundary.objects.create(utility=test_utility)
         cls.boundary_5 = Boundary.objects.create(utility=test_utility)
 
+        # Submissions activities should follow
+        # Draft-->Submission-->Approved/Revisions flow
+        # Order of test data activities dependent on order of below methods executing,
+        # So there is potentional for order variation if unexpected execution sequence
```

Comment:
```suggestion
        # So there is potential for order variation if unexpected execution sequence
```

Reply:
Only minorly embarrassed this got me twice 🤦‍♀️